codec_specs.md

```diff
@@ -88,6 +88,10 @@ The values of `BlockAddID` that are 2 of greater have no semantic meaning, but s
 associate the `BlockMore Element` with a `BlockAdditionMapping` of the associated Track.
 See (#block-additional-mapping) on Block Additional Mappings for more information.
 
+The value of 4 for `BlockAddID` (so also `BlockAddIDValue`) **SHOULD NOT** be used when `BlockAddIDType` is not 4,
```

Comment:
Maybe instead of

> The value of 4 for `BlockAddID` (so also `BlockAddIDValue`) **SHOULD NOT** be used when `BlockAddIDType` is not 4

you could simply say

> It is recommended to not use the value of 4 for `BlockAddID` (and by extension `BlockAddIDValue`) when `BlockAddIDType` is not 4

To avoid using the SHOULD NOT keyword.

Reply:
OK with that, although it should be `**RECOMMENDED**` as it's a normative term in the case.